creator-node/nginx_conf/nginx.conf

```diff
@@ -76,10 +81,14 @@ http {
             # Set client IP as `X-Forwarded-For` response header
             proxy_set_header X-Forwarded-For $remote_addr;
 
-            # Never serve a stale response
+            # proxy_cache_use_stale + proxy_cache_background_update -> deliver stale content when client requests
+            # an item that is expired or in the process of being updated from origin server
+            proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
+            proxy_cache_background_update on;
 
-            # Cache responses with 200 status, for 1 second
-            proxy_cache_valid 200 1s;
+            # Cache responses for 5second on 200, 1 second otherwise
+            proxy_cache_valid 200 5s;
```

Comment:
I dont think we should be bumping the cache for health check cus it can give us false positive but lets see if this extra 4s bump changes how the system reacts

Reply:
its already cached in cloudflare for 10 seconds, which i think i should reduce, but yeah fair i'll reduce to 2 seconds here and in CF

Reply:
reduced to 1s